solvers/qpSpline/CMakeLists.txt

```diff
@@ -1,26 +1,24 @@
-if (eigen3_FOUND)
-  add_library(drakeSplineGeneration splineGeneration.cpp SplineInformation.cpp ValueConstraint.cpp ContinuityConstraint.cpp)
-  target_link_libraries(drakeSplineGeneration drakePolynomial drakePiecewisePolynomial)
-  pods_install_libraries(drakeSplineGeneration)
-  pods_install_headers(splineGeneration.h ValueConstraint.h ContinuityConstraint.h DESTINATION drake)
-  pods_install_pkg_config_file(drakeSplineGeneration
-    LIBS -ldrakeSplineGeneration
-    REQUIRES
-    VERSION 0.0.2)
+include_directories (${CMAKE_SOURCE_DIR}/util)
+include_directories (${CMAKE_SOURCE_DIR}/solvers/polynomial)
+include_directories (${CMAKE_SOURCE_DIR}/systems/trajectories)
 
+add_library(drakeSplineGeneration splineGeneration.cpp SplineInformation.cpp ValueConstraint.cpp ContinuityConstraint.cpp)
+target_link_libraries(drakeSplineGeneration drakePolynomial drakePiecewisePolynomial)
+pods_install_libraries(drakeSplineGeneration)
+pods_install_headers(splineGeneration.h ValueConstraint.h ContinuityConstraint.h DESTINATION drake)
+pods_install_pkg_config_file(drakeSplineGeneration
+  LIBS -ldrakeSplineGeneration
+  REQUIRES
+  VERSION 0.0.2)
+
+if (MATLAB_FOUND)
   add_mex(twoWaypointCubicSplinemex twoWaypointCubicSplinemex.cpp)
-  include_directories (${CMAKE_SOURCE_DIR}/util)
-  include_directories (${CMAKE_SOURCE_DIR}/solvers/polynomial)
-  include_directories (${CMAKE_SOURCE_DIR}/systems/trajectories)
```

Comment:
Moved them above.  In fact i don't know how the drakeSplineGeneration was working without them